src/pages/_app.js

```diff
@@ -57,6 +58,21 @@ export default function App({ Component, pageProps }) {
           <Footer />
         </main>
       </ThemeProvider>
+      <Script
+        strategy="lazyOnload"
```

Comment:
Please change to `strategy="afterInteractive"`.

Reply:
Wilco

Reply:
Did